Web interface documentation (#112)
meilisearch/documentation: advanced_guides/web_interface.md

# Web interface

> After adding documents to your MeiliSearch it is possible to try out the search engine with the integrated web interface.

Reviewer: ```suggestion
After adding documents to your MeiliSearch, it is possible to try out the search engine with the integrated web interface.
```

> The **web interface is located on the same address and port as meilisearch**.

Reviewer 2: ```suggestion
The web interface is served on **the same port** as MeiliSearch.
```

> 

Reviewer: ```suggestion
```

## Example

The server is listening on: `http://127.0.0.1:7700`.

> Lets add some movies.

Reviewer: ```suggestion
Let's add some movies.
```

```bash
curl -X POST  'http://127.0.0.1:8080/indexes/movies_uid/documents'\
     --data @movies.json
```

> Lets travel to `http://127.0.0.1:7700` in our browser.

Reviewer: ```suggestion
Let's travel to `http://127.0.0.1:7700` in our browser.
```

Reviewer: And "travel"? Maybe "go"?
